SwiftUI/MyRWTutorial/SceneDelegate.swift

```diff
@@ -38,9 +38,11 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
     
     // Use a UIHostingController as window root view controller
-    let window = UIWindow(frame: UIScreen.main.bounds)
-    window.rootViewController = UIHostingController(rootView: ContentView())
-    self.window = window
-    window.makeKeyAndVisible()
+    if let windowScene = scene as? UIWindowScene {
+        let window = UIWindow(windowScene: windowScene)
+        window.rootViewController = UIHostingController(rootView: ContentView())
+        self.window = window
+        window.makeKeyAndVisible()
```

Comment:
Shouldn’t this use 2 space indentation?

Reply:
Yes, it should. Can you submit a PR to fix it? Or @bmorefield can you submit a PR to fix it?

Reply:
@rcritz Fixed indentation and submitted another PR.